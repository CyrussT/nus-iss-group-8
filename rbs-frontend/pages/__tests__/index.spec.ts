import { describe, it, expect } from 'vitest';
import { mount } from '@vue/test-utils';
import index from '../index.vue';

describe('index', () => {
  it('renders correctly', () => {
    const wrapper = mount(index, {
      global: {
        stubs: {
          UButton: true,
          UIcon: true
        }
      }
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
