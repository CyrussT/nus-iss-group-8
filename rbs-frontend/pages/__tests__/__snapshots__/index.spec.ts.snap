// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`index > renders correctly 1`] = `"<div data-v-02281a80="" class="full-page"><img data-v-02281a80="" src="/pages/image/index2.jpg" alt="Background" class="background-image"></div>"`;
